README.md

```diff
@@ -72,14 +72,21 @@ Other recommended settings
 
 Since `hy-mode` is a lisp like mode, it can use the goodness from some
 minor modes used for lisp such as
-[smartparens](https://github.com/Fuco1/smartparens) or paredit minor
+[smartparens](https://github.com/Fuco1/smartparens), 
+[RainbowDelimiters](https://github.com/Fanael/rainbow-delimiters) or paredit minor
```

Comment:
could you move this line ie. rainbow delim above.. bascially the line should read smartparens or paredit minor modes...